js/main.js

```diff
@@ -186,14 +186,6 @@ function startNeverEndingGame (images) {
     }
   })
 
-  game.afterCycle(function () {
```

Comment:
You could also consider removing the `livesLeft` variable and `resetGame` function for complete cleanup, but maybe not worth spending time on this.

Reply:
Yes, why not, I removed some unused things in 227aeb0.

In general, for a while I was hoping that our changes would be only some minor changes on top of the original codebase. But I think that ship has sailed.